hathor/p2p/manager.py

```diff
@@ -795,3 +790,17 @@ def reload_entrypoints_and_connections(self) -> None:
         self.log.warn('Killing all connections and resetting entrypoints...')
         self.disconnect_all_peers(force=True)
         self.my_peer.reload_entrypoints_from_source_file()
+
+    def get_peers_whitelist(self) -> list[PeerId]:
```

Comment:
Missing docstring in all new methods in this file.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1165/commits/b9b84ea0802ea3ae98dbc324d8ab07434a2f3575